core/jazz_deployments/index.js

```diff
@@ -67,7 +70,7 @@ function handler(event, context, cb) {
 
 			if (method === "POST" && Object.keys(path).length) {
 				logger.info("GET Deployment details using deployment Id :" + path.id);
-				exportable.processDeploymentRebuild(config, path.id, deploymentTableName)
+				exportable.processDeploymentRebuild(config, event, path.id, deploymentTableName)
```

Comment:
send event.principalId instead of sending event object.

Reply:
done